packages/commonwealth/client/scripts/views/components/AddToHomeScreenPrompt/AddToHomeScreenPrompt.scss

```diff
@@ -0,0 +1,214 @@
+@import '../../../../styles/shared';
```

Comment:
after restructuring, this file probably won't be necessary

Reply:
This errored after restructuring when i didn't include the import, so keeping it as it is for the moment.